src/collection.js

```diff
@@ -77,15 +77,20 @@ export default class Collection {
   /**
    * Retrieves the total number of records in this collection.
    *
+   * @param  {String} [since]           The since ETag param (optional).
    * @param  {Object} [options={}]      The options object.
    * @param  {Object} [options.headers] The headers object option.
    * @return {Promise<Number, Error>}
    */
-  getTotalRecords(options={}) {
+  getTotalRecords(since, options={}) {
```

Comment:
I think we should follow what was done in `listRecords()` : make `since` an option. Plus, it wouldn't break the API!